cmd/process-agent/main_common.go

```diff
@@ -310,5 +310,5 @@ func enableProfiling(cfg *config.AgentConfig) error {
 
 	v, _ := version.Agent()
 
-	return profiling.Start(cfg.ProfilingAPIKey, site, cfg.ProfilingEnvironment, "process-agent", fmt.Sprintf("version:%v", v))
+	return profiling.Start(cfg.ProfilingAPIKey, site, cfg.ProfilingEnvironment, "process-agent", profiling.DefaultProfilingPeriod, fmt.Sprintf("version:%v", v))
```

Comment:
Not a big deal but I'm curious why this only takes the default instead of the config value?

Reply:
+1

Reply:
That's because there is no config value for this agent. The config above is for system-probe, not the process-agent. I didn't want to over complicate things and add configuration parameters everywhere. I can do it if you point me where you config usually is :)

Reply:
I added an entry in the `AgentConfig` struct. @shang-wang could you double check that I updated everything that needed to be updated ? thanks